components/progress/__tests__/__snapshots__/demo.test.js.snap

```diff
@@ -849,7 +849,7 @@ Array [
           stroke="url(#ant-progress-gradient-13)"
           stroke-linecap="round"
           stroke-width="6"
-          style="stroke:[object Object];stroke-dasharray:295.3097094374406px 295.3097094374406px;stroke-dashoffset:-0px;transition:stroke-dashoffset .3s ease 0s, stroke-dasharray .3s ease 0s, stroke .3s, stroke-width .06s ease .3s, opacity .3s ease 0s"
```

Comment:
Some other commits are mixed in, please wait for #30556 to be merged, and then rebase this.

Reply:
@yoyo837 Please check now.

Reply:
You have not do `rebase`.